app/src/org/commcare/android/framework/CommCareActivity.java

```diff
@@ -70,8 +69,6 @@
     private static final String KEY_PROGRESS_DIALOG_FRAG = "progress-dialog-fragment";
     private static final String KEY_ALERT_DIALOG_FRAG = "alert-dialog-fragment";
 
-    private boolean mBannerOverriden = false;

```

Comment:
I really don't understand how this field was being used in `updateCommCareBanner`. Pretty sure there was no need for it.